fn main() {
	panic!("error occured");
	println!("Hello, world!");
}
